edi_oca/models/edi_backend.py

```diff
@@ -179,6 +179,21 @@ def _get_exchange_type_domain(self, code):
             ("backend_id", "=", self.id),
         ]
 
+    def _get_job_delay_params(self, exchange_record):
+        params = {}
+        channel = exchange_record.type_id.job_channel_id
+        if channel:
+            params["channel"] = channel.complete_name
+        return params
+
+    def _delay_action(self, rec):
+        # TODO: Remove this on 16.0
```

Comment:
Can you please log a deprecation warning?
People should switch to the new approach as soon/far as possible.